front_end/src/components/searchResultOverview/searchResultOverview.js

```diff
@@ -154,4 +137,8 @@ class searchResultOverview extends Component {
   }
 }
 
-export default searchResultOverview;
+const mapStateToProps = state => ({ result: state.searchResult });
+export default connect(
+  mapStateToProps,
+  { displayResultsOverview }
+)(searchResultOverview);
```

Comment:
2
Here you can inject 'styles' by replacing (searchResultOverview) with (withStyles(styles)(searchResultOverview));

Reply:
injected!